backend/prisma/migrations/20240627173035_workout_info/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateTable
+CREATE TABLE "Exercise" (
+    "id" SERIAL NOT NULL,
+    "workoutId" INTEGER NOT NULL,
+    "name" TEXT NOT NULL,
+
+    CONSTRAINT "Exercise_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Set" (
+    "id" SERIAL NOT NULL,
+    "exerciseId" INTEGER NOT NULL,
+    "weight" INTEGER NOT NULL,
+    "reps" DOUBLE PRECISION NOT NULL,
+
+    CONSTRAINT "Set_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "Exercise" ADD CONSTRAINT "Exercise_workoutId_fkey" FOREIGN KEY ("workoutId") REFERENCES "Workout"("id") ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
nit: use ON DELETE RESTRICT so it is safety?

Reply:
I think I want it to cascade though right? So if I delete a workout then the exercises associated are gone too. It might be a misnomer that I called the table Exercise, as it is not the database of actual exercises that will be integrated with the project.